bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/EntryLogManagerForSingleEntryLog.java

```diff
@@ -211,9 +211,7 @@ void flushRotatedLogs() throws IOException {
 
     @Override
     public void close() throws IOException {
-        if (activeLogChannel != null) {
-            activeLogChannel.close();
-        }
+        IOUtils.close(log, activeLogChannel);
```

Comment:
why did you change this? now there is no difference between close() and forceClose() method

Reply:
you are right. will revert it back